harmoni_dialogues/harmoni_bot/requirements.txt

```diff
@@ -0,0 +1 @@
+rasa==2.7.1
```

Comment:
Why the Rasa version differs between the requirements (2.7.1) file and the docker images (2.4.2)?

Reply:
Sorry, the dockerfiles have been updated to 2.7.1 now!

Reply:
Sounds good